### ISSUE TITLE
Notification message are no more found with VS Code 1.44.0

### ISSUE THREAD

**Issue by apupier** [redhat-developer/vscode-extension-tester#95]
at least, i suspect it given errors on https://travis-ci.org/github/jboss-fuse/vscode-atlasmap/builds/672863406#L716-L719

correspondign to https://github.com/jboss-fuse/vscode-atlasmap/blob/481005a637e3f5a233d87a2d126ffff719b94510/src/ui-test/basic-test.ts#L17-L26

and especially to https://github.com/jboss-fuse/vscode-atlasmap/blob/481005a637e3f5a233d87a2d126ffff719b94510/src/ui-test/common/conditions.ts#L4-L19

```
const center = await new Workbench().openNotificationsCenter();
const notifications = await center.getNotifications(NotificationType.Any);
```

**Comment by apupier** [comment]
Failing tests which seems related https://travis-ci.com/github/redhat-developer/vscode-extension-tester/jobs/318062241#L447-L451

**Comment by jrichter1** [comment]
It looks like your test code begins by calling a command from the command palette, which doesn't work in vscode 1.44.x (the fix only just landed in master). I'm guessing the command needs to be completed successfully to display the notification. 

In other words, I think the problem will go away by fixing the command execution.

**Comment by apupier** [comment]
fixed in 2.6.0
